web/src/components/tasks/IntercomScript.tsx

```diff
@@ -0,0 +1,32 @@
+import { BusinessUser } from "@businessnjgovnavigator/shared/businessUser";
+import { OperatingPhaseId } from "@businessnjgovnavigator/shared/operatingPhase";
+import Script from "next/script";
+import { ReactElement, useEffect } from "react";
+
+export const IntercomScript = ({
+  user,
+  operatingPhaseId,
+}: {
+  user?: BusinessUser;
+  operatingPhaseId?: OperatingPhaseId;
+}): ReactElement => {
+  useEffect(() => {
+    if (!operatingPhaseId) return;
+    const item = document.querySelector("#intercom-identity");
+    item?.setAttribute("data-user-operating-phase", operatingPhaseId);
+  }, [operatingPhaseId]);
```

Comment:
What is the purpose of this useEffect?

Reply:
Because <Script> is a next component, it wasn't re-rendering after initial render, so we have to manually edit/modify it's dom element attribute